app/controllers/supplejack_api/harvester/records_controller.rb

```diff
@@ -90,7 +90,8 @@ def index
         return render status: 400, body: body if search_params.blank?
 
         page = search_options_params[:page].to_i
-        @records = SupplejackApi.config.record_class.where(search_params).page(page).per(20)
+
+        @records = SupplejackApi.config.record_class.where(search_params).page(page).per(20).hint(hints)
```

Comment:
I have not written any tests for this, because the indexes are not on the gem, they are on the host api.   As such it's very hard to test